DEV-9/EntryPoint.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace ReplacementOfCharacterGroups
+{
+  class EntryPoint
+  {
+    static void Main(string[] args)
+    {
+      try
+      {
+        List<string> seqList = new FileReader().ReadLines(@"c:\Text.txt");
+        string firstSequence = seqList[0];
```

Comment:
no check for the size of seqList, the catch of the exception say nothing about the issue